### PR TITLE
ProductCard title link

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

Wrapped the title and subtitle text in a link.

Improved the Product card story.

## Justify why they are needed

[Notion page](https://www.notion.so/hedviginsurance/Make-the-headline-text-in-Product-grid-as-the-link-text-inside-a-a-not-L-s-mer-OK-if-the-d0e2b5f78a6a473693834d73a43b9c79?pvs=4)

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Review by robinandeer — CHANGES_REQUESTED** [review]
Now this component is getting really confusing with three clickable elements that all do the same thing 😅 

I would not consider If website the best standard - they use some image with text inside it 😛 

I think we should do what apple does: https://www.apple.com/

1. Wrap the WHOLE component in a `<Link>` with `tabIndex={-1} aria-hidden={true}`
2. Use a `<ButtonNextLink>` for "read more"

**Comment by robinandeer** [comment]
@KarnellSchultz sorry realized just now that Apple has a more advanced setup - "a" should never be nested with "a"

But same goes for nesting "button" inside "a" -- we need another solution for that

**Comment by KarnellSchultz** [comment]
@robinandeer Thanks for the tips. Going to play around with the component for a bit to see if I can get

File: apps/store/src/components/ProductCard/ProductCard.tsx [hunk 1]
```diff
@@ -62,6 +66,7 @@ const ImageWrapper = styled.a<ImageSize>(({ aspectRatio }) => ({
 
   ':hover, :active': {
     opacity: 0.95,
+    transition: 'opacity 0.1s ease-out',
```
Comment (KarnellSchultz) [inline apps/store/src/components/ProductCard/ProductCard.tsx:69]: matching the buttons transition

**Comment by KarnellSchultz** [comment]
Screenshot 2023-02-14 at 12 21 49

File: apps/store/src/components/ProductCard/ProductCard.tsx [hunk 2]
```diff
@@ -29,26 +29,30 @@ export const ProductCard = ({
...
+          {t('READ_MORE')}
+        </ButtonNextLink>
+      </CallToAction>
+    </Wrapper>
   )
 }
 
-const ImageWrapper = styled.a<ImageSize>(({ aspectRatio }) => ({
+const Wrapper = styled(Space)({
+  height: '100%',
```
Comment (robinandeer) [inline apps/store/src/components/ProductCard/ProductCard.tsx:52]: I think I removed this to fix an issue in Safari 😅

**Review by robinandeer — APPROVED** [review]
Very nice!

Just make sure the design is not broken in safari with "height = 100%"

File: apps/store/src/components/ProductCard/ProductCard.tsx (hunk 2 above)
Comment (KarnellSchultz) [reply apps/store/src/components/ProductCard/ProductCard.tsx]: Removed it and (re)fixed the layout 👍

**Pull request merged**